index.js

```diff
@@ -74,6 +78,8 @@ function togpx( geojson, options ) {
   else
     features = [{type:"Feature", properties: {}, geometry: geojson}];
   features.forEach(function mapFeature(f) {
+    if (!f.hasOwnProperty('properties'))
+      f.properties = {};
```

Comment:
I tried to prevent runtime errors for the case `f.properties` isn't defined. In that case I prefer one input check at the beginning only, instead of do multiple checks later. 
But your reasoning about accepting valid input only is another way of looking at it. Okay to me.